core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -188,4 +208,35 @@ class BillingProfileManagerDAOImpl(
     } yield billingProjects.filter(bp => samUserResourceIds.contains(bp.projectName.value))
   }
 
+  private def getProfileApiPolicy(samRole: ProjectRole): String =
+    samRole match {
```

Comment:
I thought this was better than lower-casing ProjectRoles.Owner.toString and ProjectRoles.User.toString, as it doesn't rely on knowledge of what those values happen to be.